pkgs/applications/graphics/ImageMagick/default.nix

```diff
@@ -122,7 +122,7 @@ stdenv.mkDerivation (finalAttrs: {
     (cd "$dev/include" && ln -s ImageMagick* ImageMagick)
     moveToOutput "bin/*-config" "$dev"
     moveToOutput "lib/ImageMagick-*/config-Q16HDRI" "$dev" # includes configure params
-    cp $src/www/source/configure.xml $out/share/ImageMagick-*/
+    grep -v '/nix/store' $dev/lib/ImageMagick-*/config-Q16HDRI/configure.xml > $out/share/ImageMagick-7/configure.xml
```

Comment:
Are you sure that path stays the same?

Reply:
Thanks, I've corrected this in the squashed commit.

Reply:
I was also referring to `Q16HDRI`.

Reply:
That was hardcoded in the derivation prior to my PR (the `moveToOutput` line just prior to my addition).

Doing a quick search through the source code it doesn't seem to be set anywhere, although it is used in multiple places.

Reply:
> Are you sure that path stays the same?

It is hard-coded into several files and hasn't changed in several years. 
I think it'd be a good idea to comment this part of the code briefly though.